### ISSUE TITLE
Auto using lvl.100 ships for lvl. 50 commisions

### ISSUE THREAD

**Issue by mrminht2000** [LmeSzinc/AzurLaneAutoScript#1629]
### Is your feature request related to a problem?

Alas always uses Advice feature when chooses the ships for the commisions. The problem is when there are some lvl. 50 commisions, which require at least one lvl. 100 or above ship, the Advice button can't pick any of my ships although lots of them are above the requirement. 
So that I usually have one or two fleets that aren't used for the commisions when lvl. 50 commisions appear.
I did searched for the solution, removed lvl.100 ship from every fleets (Sortie, OpSi and Exercise), unequiped  their equipment, ... but it's never worked.

### Describe the solution you'd like

I think it would better if the commisions feature would have something that could choose the ship when Advice button didn't worked.

**Comment by llllllllallll** [comment]
This may be because your ship is in a difficult formation

**Comment by mrminht2000** [comment]
> This may be because your ship is in a difficult formation

Nope, I used to use Ayanami in hard-mode. Then I did remove her from every hard-mode maps, normal fleet, OpSi and Exercise, but it still didn't work

**Comment by Luiso9** [comment]
BUMP, i have the same problem... been trying to avoid Commisions lvl. 50 or above with filter but also didnt work.
